src-utils/HydraAuctionUtils/Tx/AutoCreateTx.hs

```diff
@@ -0,0 +1,201 @@
+{-# LANGUAGE RecordWildCards #-}
```

Comment:
@mlabs-haskell/hydra-auction 

That is moved file!

Most of stuff are the same. All logical changes you can see in separate `commit` before moving